pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -764,6 +764,14 @@ func (m *kubeGenericRuntimeManager) SyncPod(ctx context.Context, pod *v1.Pod, po
 		// When runc supports slash as sysctl separator, this function can no longer be used.
 		sysctl.ConvertPodSysctlsVariableToDotsSeparator(pod.Spec.SecurityContext)
 
+		// Prepare resources for pod containers
+		for _, container := range append(pod.Spec.InitContainers[:], pod.Spec.Containers...) {
+			err := m.runtimeHelper.PrepareDynamicResources(pod, &container)
+			if err != nil {
+				return
+			}
+		}
+
```

Comment:
1. Looks like the `[:]` is still there.

2. If all we do is loop across containers here to call `PrepareDynamicResources(pod, &container)`, is there any reason to not have the API (all the way down into the draManager) just be `PrepareDynamicResources(pod)`. That would make it more symmetrical with the ÙnprepareDynamicResources(pod)` call, which doesn't take a container.

Reply:
done. Please, review again.